tests/foreman/ui/test_jobinvocation.py

```diff
@@ -27,6 +32,77 @@ def module_rhel_client_by_ip(module_org, smart_proxy_location, rhel7_contenthost
     return rhel7_contenthost
 
 
+@pytest.mark.no_containers
```

Comment:
This seems to be redundant here since you are not using a contenthost fixture in the test.